receiver/prometheusreceiver/internal/util_test.go

```diff
@@ -138,23 +138,17 @@ func TestGetBoundary(t *testing.T) {
 			wantValue: 11.71,
 		},
 		{
-			name:    "summary with bucket label",
```

Comment:
Why remove these test cases?

Reply:
I think their intent was to show that a Summary needed quantile labels (and not bucket labels Histograms have) to be valid. As I'm removing the requirement, the test can just check if no labels at all is fine. The other test is an accidental duplicate I think, it does the exact same thing.

Reply:
I think we still want to at least log errors if we get a series that isn't sum or count without a quantile label.  I'd like to keep this case if possible.